scalar/interval.go

```diff
@@ -64,15 +70,12 @@ func (s *MonthInterval) Set(value any) error {
 		s.Value = int64(mi.Months)
 		return nil
 	case *string:
-		if v == nil {
-			return s.Int.Set(nil)
```

Comment:
Can we split this fix into a separate PR?

Reply:
@erezrokah https://github.com/cloudquery/plugin-sdk/pull/1477